CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.2.1
+
+Major:
+  * Infrastructure for testing coroutine-specific code in `kotlinx-coroutines-test`: `runBlockingTest`, `TestCoroutineScope` and `TestCoroutineDispatcher`, contributed by Sean McQuillan (@objcode).
+  * `Job.asCompletableFuture` extension in jdk8 module (#1113).
+
+Flow improvements:
+  * `flowViaChannel` rework: block parameter is no longer suspending, but provides `CoroutineScope` receiver and allows conflated channel (#1081, #1112).
+  * New operators: `switchMap`, `sample`, `debounce` (#1107).
+  * `consumerEach` is deprecated for removal on `Publisher`, `ObservableSource` and `MaybeSource`, `collect` extension is introduced instead (#1080).
```

Comment:
"is deprecated" is enough :) We're not in Java. In Kotlin if its deprecated it will be hidden or removed.